app/javascript/dashboard/routes/dashboard/settings/automation/AddAutomationRule.vue

```diff
@@ -238,8 +256,44 @@ export default {
   .info-message {
     font-size: var(--font-size-mini);
     color: var(--s-500);
-    text-align: right;
   }
   margin-bottom: var(--space-medium);
 }
+
+.row {
+  height: 100%;
+}
+.canvas {
+  background-image: radial-gradient(var(--s-75) 1.2px, transparent 0);
```

Comment:
Since we are almost repeating the DOM and same styles in two files, Can we create a dumb component which can be used in this two files?

Reply:
@nithindavid Can I do this in a new PR?

Dont want to make it too big to review.

CC: @muhsin-k